pjsip/src/pjsua2/account.cpp

```diff
@@ -96,7 +96,7 @@ void RtcpFbConfig::readObject(const ContainerNode &node) PJSUA2_THROW(Error)
         NODE_READ_NUM_T         (cap_node, pjmedia_rtcp_fb_type, cap.type);
         NODE_READ_STRING        (cap_node, cap.typeName);
         NODE_READ_STRING        (cap_node, cap.param);
-        this->caps.push_back(cap);
+        this->caps.push_back(std::move(cap));
```

Comment:
General for all occurency:

std::move is a code smell. You still have two objects.

With C++11 you can use:
```
....emplace_back([&](){
   RtcpFbCap cap;
   // initialize
   return cap;
});
```
Then it the object directly, thanks to RVO.

Even better would be deligated identifier. But most probably not for the minimum wanted c++ version.